README.md

```diff
@@ -145,6 +153,165 @@ readData(dataInput) {
 }
 ```
 
+# Using Node.js Client with Hazelcast IMDG
+
+## 4. Distributed Events
+
+This chapter introduces Distributed Events which are Cluster Events and Distibuted Data Structure Events.
+
+### 4.1. Cluster Events
+
+You can add event listeners to a Hazelcast Node.js client. You can configure the following listeners to listen to the events on the client side.
+
+`Membership Listener`: Notifies when a member joins to/leaves the cluster, or when an attribute is changed in a member.
+
+`Distributed Object Listener`: Notifies when a distributed object is created or destroyed throughout the cluster.
+
+`Lifecycle Listener`: Notifies when the client is starting, started, shutting down, and shutdown.
+
+#### 4.1.1. Listening for Member Events
+
+You can add the following 3 different types of member events to the `ClusterService`.
```

Comment:
Let's not use any number when introducing ordered/unordered lists:

"You can the following types of member events....."